wire-ios-sync-engine/Source/Notifications/VoIPPushManager.swift

```diff
@@ -182,7 +182,8 @@ public final class VoIPPushManager: NSObject, PKPushRegistryDelegate {
         payload: [AnyHashable: Any],
         completion: @escaping () -> Void
     ) {
-        Self.logger.trace("process voIP push, payload: \(payload)")
+        // TODO: check this is dead code right?
```

Comment:
from my understanding this is never used, we always use the APNS code, is that right?

Reply:
The device will only receive APNS pushes and wake the NSE, but the NSE can create a "fake voip push" when it detects a call event and pass it to the main app.